node/node_newblock.go

```diff
@@ -225,17 +235,22 @@ func (node *Node) proposeNewBlock() (*types.Block, error) {
 		return nil, err
 	}
 
-	// Prepare last commit signatures
-	sig, mask, err := node.Consensus.BlockCommitSig(header.Number().Uint64() - 1)
+	finalizedBlock, err := node.Worker.FinalizeNewBlock(
+		commitSigs, node.Consensus.GetCurBlockViewID(),
+		coinbase, crossLinksToPropose, shardState,
+	)
 	if err != nil {
-		utils.Logger().Error().Err(err).Msg("[proposeNewBlock] Cannot get commit signatures from last block")
+		utils.Logger().Error().Err(err).Msg("[proposeNewBlock] Failed finalizing the new block")
 		return nil, err
 	}
+	utils.Logger().Info().Msg("[proposeNewBlock] verifying the new block header")
```

Comment:
minor `proposeNewBlock` => `ProposeNewBlock`. what is the purpose of this info msg?

Reply:
done.